src/Microsoft.ML.Tokenizers/Model/BertTokenizer.cs

```diff
@@ -0,0 +1,729 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.ML.Tokenizers
+{
+    /// <summary>
+    /// Tokenizer for Bert model.
+    /// </summary>
+    /// <remarks>
+    /// The BertTokenizer is a based on the WordPieceTokenizer and is used to tokenize text for Bert models.
+    /// The implementation of the BertTokenizer is based on the original Bert implementation in the Hugging Face Transformers library.
+    /// https://huggingface.co/transformers/v3.0.2/model_doc/bert.html?highlight=berttokenizerfast#berttokenizer
+    /// </remarks>
+    public sealed partial class BertTokenizer : WordPieceTokenizer
```

Comment:
How does this compare in behavior / performance to https://www.nuget.org/packages/FastBertTokenizer ?

Reply:
I didn't compare the perf but FastBertTokenizer is very customized to the Bert tokenizer only and our implementation is just conforming to our abstract interfaces which I can expect have some overhead. But in general, I expect our implementation should be good enough in common scenarios.